packages/server-daemon/src/services/TTS/scripts/openvoice_speak.py

```
@@ -0,0 +1,271 @@
+                        new_frame_rate = int(audio.frame_rate * speed)
+                        audio = audio._spawn(
+                            audio.raw_data, overrides={"frame_rate": new_frame_rate}
+                        )
+                        audio = audio.set_frame_rate(audio.frame_rate)
+
+                    audio.export(output_path, format="wav")
+                except ImportError:
+                    # pydub not available, skip speed adjustment
+                    pass
+
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)